pypot/server/httpserver.py

```diff
@@ -218,12 +219,23 @@ def get_motors_register_value(register_name):
 
             return registers_motors
 
-
     def run(self, quiet=None, server='tornado'):
         """ Start the bottle server, run forever. """
         if quiet is None:
             quiet = self.quiet
-        bottle.run(self.app,
-                   host=self.host, port=self.port,
-                   quiet=quiet,
-                   server=server)
+        try:
+            bottle.run(self.app,
+                       host=self.host, port=self.port,
+                       quiet=quiet,
+                       server=server)    
+        except RuntimeError as e:
```

Comment:
That's a strange behaviour no? Why does they raise a runtime error if there is no problem. Did you find any explanation on their side?